RM_vs_DFN.ipynb

```diff
@@ -0,0 +1,337 @@
+{
```

Comment:
<p>It would be good adding, after the plots, some discussion of what we observe, namely that the RM has very good agreement with DFN for all SEI models.</p><p> </p><p>It would also be good to print the citations at the end of the notebook (like with PyBaMM example notebooks).</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/brosaplanella/formation/pull/14/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>

Reply:
Some discussions and citations have been added. (but it's not clear how citations work or how to add customer citations. Some papers appear to be outdated, unlike your paper.)